src/ngx_http_lua_rewriteby.c

```diff
@@ -126,6 +126,13 @@ ngx_http_lua_rewrite_handler(ngx_http_request_t *r)
 
                 return NGX_HTTP_OK;
             }
+            else {
```

Comment:
The coding style around `} else {` is wrong. Please see similar places in the existing code base. Please be very careful :)